### PR TITLE
Remove remaining references to P2 functionality

### PR DESCRIPTION
**Pull request by sschuberth**
P2 functionality was removed in e3316f7.

Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch.io>

### CODE REVIEW

**Comment by sschuberth** [comment]
Thanks for the link, I've restored the `example-projects/p2-example-project` and `example-projects/p2-example-tycho-project` projects. But shouldn't we then also keep `.ci-scripts/test-p2-end2end.sh` as it operates on `example-projects/p2-example-tycho-project`?

**Comment by bs-ondem** [comment]
Yes we should keep the end2end test, but we can remove all parts from the bash scripts regarding the P2 Module e.g. run_all_tests.sh.

File: .github/workflows/pull_request_build.yml [hunk 1]
```diff
@@ -59,9 +59,6 @@ jobs:
     - name: Gradle Frontend Test
       run: bash .ci-scripts/test-ExampleTestProject-with-gradle.sh
 
-    - name: p2Test
```
Comment (bs-ondem) [inline .github/workflows/pull_request_build.yml]: We should keep this end2end test.

File: .travis.yml [hunk 2]
```diff
@@ -76,10 +76,6 @@ jobs:
     - name: gradle frontend test
       script: .ci-scripts/test-ExampleTestProject-with-gradle.sh
     
-    # test P2
-    - name: p2Test
-      script: .ci-scripts/test-p2-end2end.sh
```
Comment (bs-ondem) [inline .travis.yml]: We should keep this end2end test.

File: core/frontend-stubs-testing/src/main/java/org/eclipse/sw360/antenna/frontend/testing/AbstractAntennaFrontendTest.java [hunk 3]
```diff
@@ -66,8 +66,7 @@ protected void setRunExecutionTest(boolean runExecutionTest) {
                 {(Supplier<AbstractTestProjectWithExpectations>) MinimalTestProject::new, "minimal configuration"},
                 {(Supplier<AbstractTestProjectWithExpectations>) BasicTestProject::new, "basic configuration"},
                 {(Supplier<AbstractTestProjectWithExpectations>) ExampleTestProject::new, "example configuration"},
-                {(Supplier<AbstractTestProjectWithExpectations>) MavenTestProject::new, "maven configuration"},
-                {(Supplier<AbstractTestProjectWithExpectations>) P2TestProject::new, "p2 configuration"}
```
Comment (bs-ondem) [inline core/frontend-stubs-testing/src/main/java/org/eclipse/sw360/antenna/frontend/testing/AbstractAntennaFrontendTest.java]: If we keep the p2 example tests, we should also keep the Test classes for it.

File: .github/workflows/pull_request_build.yml (hunk 1 above)
Comment (sschuberth) [reply .github/workflows/pull_request_build.yml]: Done.

File: .travis.yml (hunk 2 above)
Comment (sschuberth) [reply .travis.yml]: Done.

File: core/frontend-stubs-testing/src/main/java/org/eclipse/sw360/antenna/frontend/testing/AbstractAntennaFrontendTest.java (hunk 3 above)
Comment (sschuberth) [reply core/frontend-stubs-testing/src/main/java/org/eclipse/sw360/antenna/frontend/testing/AbstractAntennaFrontendTest.java]: Done.

**Pull request merged**